test/context.test.js

```diff
@@ -49,6 +49,15 @@ describe('Context', function () {
       context = new Context(event)
       expect(context.repo()).toEqual({owner: 'bkeepers-inc', repo: 'test'})
     })
+
+    it('returns error for context.repo()', function () {
```

Comment:
nit: 'return error for context.repo() when repository doesn't exist

Reply:
done.